### PR TITLE
chore: Updating Lookout documentation

### PR DESCRIPTION
**Pull request by aeliaNR**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

Fixing Lookout and eLookout documentation. NR-78122
- Removing APM specific language; Diagnose with Lookout capability is available to [all entity types with golden signal definitions](https://github.com/newrelic/entity-definitions/)
- Correcting an error from [a prior PR](https://github.com/newrelic/docs-website/pull/12779) which removed all language that specified Lookout account requirements
- Correcting some minor style issues (e.g. - Traces tab section had the wrong level of heading)

### CODE REVIEW

File: src/content/docs/apm/apm-ui-pages/monitoring/lookout-signals.mdx [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-title: Diagnose deviations in APM signals with Lookout
+title: Diagnose anomalies with Lookout
 ---
 
 import embedded_lookout_diagnose_button from 'images/embedded-lookout-diagnose-button.webp'
 
 import embedded_lookout_detail from 'images/embedded-lookout-detail.webp'
 
-Use [New Relic Lookout](/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance/) to detect deviations in your APM service entities' three golden signals (response time, error rate, and throughput) and analyze the behavior and causes of those deviations.
+Use [New Relic Lookout](/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance/) to detect anomalies in your entities' golden signals (E.g. - response time, error rate, and throughput for a service) and analyze the behavior and causes of those anomalies.
```
Comment (ally-sassman) [inline src/content/docs/apm/apm-ui-pages/monitoring/lookout-signals.mdx:9]:
```suggestion
Use [New Relic Lookout](/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance/) to detect anomalies in your entities' golden signals (for example, the response time, error rate, and throughput for a service) and analyze the behavior and causes of those anomalies.
```

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx [hunk 2]
```diff
@@ -237,3 +213,53 @@ You can also create your own query. To do that:
...
+To visualize and analyze a specific event attribute, select show detailed view.
+
+### Related changes tab [#related-changed]
+
+New Relic Lookout can find other signals that began behaving differently around the same time as the selected signal for that entity type. Clicking the name of the signal reruns the analysis, focusing on that key performance indicator. This tab shows correlated behavior in the golden signals of related entities for a selected golden signal of the main entity. You can select the different golden signals and you can view signals which have a high confidence score or a low confidence. This tab displays a table format and, to dig in further, you can click through to the correlated entity. 
+
+<img
+  title="Related changes"
+  alt="An image showing the related changes tab"
+  src={diagnoseRelatedChanges}
```
Comment (clarkmcadoo) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx:244]:
```suggestion
  src={diagnoseRelatedChange}
```

Comment (clarkmcadoo) [reply src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx:244]: This should get the build working!

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx [hunk 3]
```diff
@@ -237,3 +213,53 @@ You can also create your own query. To do that:
...
+
+To analyze an application or facet, click a circle, table row, or abnormal golden signal. When clicked, the button opens a diagnostic tool you can use to explore details about your golden signals. The details panel shows a link to the affected [entity](/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic), the golden signals over time and deviations of the performance indicator, and recent alert and deployment activity for that entity, when available.
+
+There are up to five tabs available. The details of the individual tabs are outlined further.
+
+### Workload change tab [#workload-change]
+
+This tab helps you spot patterns in event attributes that might be related to the deviation in the main entity.
+
+You can select an event type see how the values of its event attributes changed for your entity in the selected time period against the preceding same duration. A timeline visualization is provided to show the sample time period and the preceding comparison time period.
```
Comment (ally-sassman) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx:227]:
```suggestion
This tab helps you spot patterns in event attributes that might be related to the deviation in the main entity.

You can select an event type to see how the values of its event attributes changed for your entity in the selected time period against the preceding same duration. A timeline visualization is provided to show the sample time period and the preceding comparison time period.
```

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx [hunk 4]
```diff
@@ -237,3 +213,53 @@ You can also create your own query. To do that:
...
+
+<img
+  title="Workload change"
+  alt="An image showing the workload change tab"
+  src={diagnoseWorkloadChange}
+/>
+
+To visualize and analyze a specific event attribute, select show detailed view.
```
Comment (ally-sassman) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx:235]:
```suggestion
To visualize and analyze a specific event attribute, toggle **Show detailed view**.
```

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx [hunk 5]
```diff
@@ -237,3 +213,53 @@ You can also create your own query. To do that:
...
+
+### Related changes tab [#related-changed]
+
+New Relic Lookout can find other signals that began behaving differently around the same time as the selected signal for that entity type. Clicking the name of the signal reruns the analysis, focusing on that key performance indicator. This tab shows correlated behavior in the golden signals of related entities for a selected golden signal of the main entity. You can select the different golden signals and you can view signals which have a high confidence score or a low confidence. This tab displays a table format and, to dig in further, you can click through to the correlated entity.
```
Comment (ally-sassman) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx:239]:
```suggestion
New Relic Lookout can find other signals that began behaving differently around the same time as the selected signal for that entity type. Clicking the name of the signal reruns the analysis, focusing on that key performance indicator. This tab shows correlated behavior in the golden signals of related entities for a selected golden signal of the main entity. You can select the different golden signals and you can view signals which have a high or low confidence score. This tab displays a table format and, to dig in further, you can click through to the correlated entity. 
```

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx [hunk 6]
```diff
@@ -237,3 +213,53 @@ You can also create your own query. To do that:
...
+
+<img
+  title="Related changes"
+  alt="An image showing the related changes tab"
+  src={diagnoseRelatedChange}
+/>
+
+### Seasonality tab [#seasonality]
+
+The seasonality tab shows the changes in the selected golden signal over time and also surfaces any patterns in the anomalies that we find. Once you select the golden signal you can then view the data over an hour, a day or a week. The anomaly patterns are also presented in a table below the chart and you can click on a link to see more details of the anomaly.
```
Comment (ally-sassman) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx:249]:
```suggestion
The Seasonality tab shows the changes in the selected golden signal over time and also surfaces any patterns in the anomalies that we find. Once you select the golden signal you can then view the data over an hour, a day or a week. The anomaly patterns are also presented in a table below the chart and you can click on a link to see more details of the anomaly.
```

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx [hunk 7]
```diff
@@ -237,3 +213,53 @@ You can also create your own query. To do that:
...
+
+### Performance tab [#performance]
+
+This tab shows charts for other key performance indicators for the selected application or facet. 
+
+When the target is a New Relic application, we show the top transactions, error classes, external services, and database operations, alongside links to the relevant features. If there is no data for any of these, the charts are not shown.
+
+### Traces tab [#traces]
+
+If the entity has distributed traces configured and available in the selected evaluated time window, the **Traces** section is enabled. Each card shows a summary of a trace and can be clicked for details. **Explore all traces** links to the [distributed traces application](/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui), filtered to the selected entity.
```
Comment (ally-sassman) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx:265]:
```suggestion
The Traces tab is displayed only if the entity has distributed traces configured and available in the selected evaluated time window. Each card shows a summary of a trace and can be clicked for details. **Explore all traces** links to the [distributed traces application](/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui), filtered to the selected entity.
```

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx [hunk 8]
```diff
@@ -237,3 +213,53 @@ You can also create your own query. To do that:
...
+  title="Workload change"
+  alt="An image showing the workload change tab"
+  src={diagnoseWorkloadChange}
+/>
+
+To visualize and analyze a specific event attribute, toggle **Show detailed view**.
+
+### Related changes tab [#related-changed]
+
+New Relic Lookout can find other signals that began behaving differently around the same time as the selected signal for that entity type. Clicking the name of the signal reruns the analysis, focusing on that key performance indicator. This tab shows correlated behavior in the golden signals of related entities for a selected golden signal of the main entity. You can select the different golden signals and you can view signals which have a high or low confidence score. This tab displays a table format and, to dig in further, you can click through to the correlated entity.
```
Comment (ally-sassman) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx:239]:
```suggestion
The Related changes tab shows correlated behavior in the golden signals of related entities for a selected golden signal of the main entity. New Relic Lookout can find other signals that began behaving differently around the same time as the selected signal for that entity type. Clicking the name of the signal reruns the analysis, focusing on that key performance indicator. You can select the different golden signals and you can view signals which have a high or low confidence score. This tab displays a table format and, to dig in further, you can click through to the correlated entity.
```

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx [hunk 9]
```diff
@@ -237,3 +213,53 @@ You can also create your own query. To do that:
...
+  title="Seasonality"
+  alt="An image showing the seasonality tab"
+  src={diagnoseSeasonality}
+/>
+
+### Performance tab [#performance]
+
+The Performance tab shows charts for other key performance indicators for the selected application or facet. 
+
+When the target is a New Relic application, we show the top transactions, error classes, external services, and database operations, alongside links to the relevant features. If there is no data for any of these, the charts are not shown.
```
Comment (ally-sassman) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx:261]:
```suggestion
The Performance tab shows charts for other key performance indicators for the selected application or facet. When the target is a New Relic application, we show the top transactions, error classes, external services, and database operations, alongside links to the relevant features. If there is no data for any of these, the charts are not shown.
```

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx [hunk 10]
```diff
@@ -237,3 +213,53 @@ You can also create your own query. To do that:
...
+
+<img
+  title="Workload change"
+  alt="An image showing the workload change tab"
+  src={diagnoseWorkloadChange}
+/>
+
+To visualize and analyze a specific event attribute, toggle **Show detailed view**.
+
+### Related changes tab [#related-changed]
```
Comment (ally-sassman) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx:237]:
```suggestion
### Related changes tab [#related-changes]
```

**Review by ghost — APPROVED** [review]

**Pull request merged**